lib/LaTeXML/resources/javascript/LaTeXML-maybeMathjax.js

```diff
@@ -1,7 +1,7 @@
 //======================================================================
 // Load MathJax, IFF the current browser can't handle MathML natively.
 
-(function() {
+(function () {
     var mathjax_url =
         "https://cdn.jsdelivr.net/npm/mathjax@3/es5/tex-mml-chtml.js?config=MML_HTMLorMML";
```

Comment:
A semi-related comment: I think MathJax 3 changed the URL conventions. E.g. I directly download `es5/mml-chtml.js` in my templates, because it is about 160k smaller. The `config=` line sounds like a leftover MathJax 2?